resources/pod.py

```diff
@@ -112,7 +129,9 @@ def exec_ceph_cmd(self, ceph_cmd):
             raise CommandFailed(
                 "Ceph commands can be executed only on toolbox pod"
             )
-        ceph_cmd += " --format json-pretty"
+        ceph_prefix = "ceph "
```

Comment:
@ebenahar there are some ceph commands which do not start with the prefix "ceph". So wouldn't it be better to not have a prefix defined and keep a generic function to run all kind of ceph related commands in toolbox pod ?
One e.g
**# rbd ls -p rbd**
pvc-4273b6de-8ba9-11e9-a45f-02f0cff6daa2

Reply:
thanks, done